src/components/pages/AdminHome/ApplicationCard.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card } from 'antd';
+import { connect, useDispatch } from 'react-redux';
+import { getInstructors } from '../../../redux/actions/instructorActions';
+import { useOktaAuth } from '@okta/okta-react';
+import '../../../styles/AdminDashboardHome/index.less';
+
+const ApplicationCard = props => {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { authState } = useOktaAuth();
+
+  useEffect(() => {
+    if (!authState) return;
+    dispatch(getInstructors(authState.idToken.idToken));
+  }, [authState]);
+
+  const handleOnClick = id => {
+    history.push(`/instructors/${id}`); // Might have to change the route.
+  };
+
+  const sorted = props.instructor.instructors.sort(
+    (a, b) => new Date(b.date) - new Date(a.date)
+  );
```

Comment:
I am curious to know if the sorted function has been tested. The way I read it, it appears to be creating two new dates at the time the function is called, which may not be sorting the cards the way it's intended. I have not tested it myself, so I could be wrong.

Reply:
We tested it with dummy data and multiple different dates. its working as intended

Reply:
Would be cool to have a query string on the API call for instructors that would automatically sort it for us in the response. Just something I was thinking about. Great work!